Update docs for RN SDK v5 (#6230)
getsentry/sentry-docs: src/platforms/react-native/manual-setup/manual-setup.mdx

description: "Learn about manual configuration for iOS and Android."
---

> If you can’t (or prefer not to) run the [automatic setup](/platforms/react-native), you can use the instructions below to configure your application.

Reviewer: ```suggestion
If you can’t (or prefer not to) run the [automatic setup](/platforms/react-native), use the instructions below to configure your application.
```

<Alert level="info" title="Prerequisite">

> Before you begin, you need to have already created a `sentry.properties` file at the root of your React Native _android_ and _ios_ project directories. An example of this properties file for [iOS](https://github.com/getsentry/examples/blob/master/react-native/AwesomeProject/ios/sentry.properties) and [Android](https://github.com/getsentry/examples/blob/master/react-native/AwesomeProject/android/sentry.properties).
> 

Reviewer: ```suggestion
Before you can begin, you need to created a `sentry.properties` file at the root of your React Native _android_ and _ios_ project directories. Check out these examples of property files for [iOS](https://github.com/getsentry/examples/blob/master/react-native/AwesomeProject/ios/sentry.properties) and [Android](https://github.com/getsentry/examples/blob/master/react-native/AwesomeProject/android/sentry.properties).

```

</Alert>

## iOS

### Linking the Native Library via CocoaPods

> You must link to the native module in your Podfile. In React Native versions 0.60 and above the podspec should be automatically linked, if that's not the case you can manually link the native module by adding the `RNSentry.podspec` to your podfile like below:

Reviewer: ```suggestion
To link the Native library via CocoaPods, link to the Native module in your Podfile. In React Native versions 0.60 and above, the podspec should be automatically linked, if that's not the case, you can link to the native module manually by adding the `RNSentry.podspec` to your Podfile. See below:
```

}
```

> If you are linking manually make sure your `MainApplication.java` is similar to:

Reviewer: ```suggestion
If you're linking manually, make sure your `MainApplication.java` is similar to:
```
